website/docs/r/postgresql_schema.html.markdown

```diff
@@ -57,6 +57,7 @@ resource "postgresql_schema" "my_schema" {
 
 * `name` - (Required) The name of the schema. Must be unique in the PostgreSQL
   database instance where it is configured.
+* `database` - (Optional) The DATABASE in which where this schema will be created. (Default: postgres)
```

Comment:
> I assumed that it would use the provider configuration to infer this

Actually your assumption was right, if database is not specified in `postgresql_schema`, it creates the schema in the database configured in the provider.

I've just tested it to be sure.

Reply:
hey @cyrilgdn thank you for looking into this. Did you try with a database name that is not `postgres`? I used a different database name (`test_calvin`) in my provider configuration and it defaults to the `postgres` database actually unless I explicitly specify the database field

Reply:
then again I might be doing something incorrectly. Should the Default wording be updated to reflect the database used by your provider configuration?

Reply:
Sorry, the one thing I should add is that I’m using AWS Aurora Postgres and I don’t have access to a super user

Reply:
Apparently less sleep and Terraform configuration don't mix 🤣 
You are correct, I was not specifying the database in my provider so it was defaulting to `postgres`